src/main/java/org/opentripplanner/graph_builder/module/AddTransitModelEntitiesToGraph.java

```diff
@@ -107,8 +107,7 @@ private void addStopsToGraphAndGenerateStopVertexes(Graph graph) {
     }
 
     private void addStationsToGraph(Graph graph) {
-        // TODO OTP2 MMSP - What does 'linked' means
-        // Store parent stops in graph, even if not linked.
+        // Store parent stops in graph, even if not connected to the road network
```

Comment:
This is always true, so we should delete the comment. Stations are not part of the street graph any more.